src/controllers/trip.controller.js

```diff
@@ -39,27 +39,42 @@ class tripController {
       const status = 'pending';
       const { type } = req.body;
 
-      const createdTrip = await tripService.CreateTrip(req, req.body, tripid, type);
+      const createdTrip = await tripService.CreateTrip(
+        req,
+        req.body,
+        tripid,
+        type
+      );
 
       const { tripId } = createdTrip;
 
       const userId = req.user.id;
 
-
       const userManager = await userService.findUserManager(userId);
       const { managerId } = userManager[0];
       const requestData = {
-        tripId, userId, managerId, status
+        tripId,
+        userId,
+        managerId,
+        status
```

Comment:
Unnecessary change. I think we might have the previous code we don't need to change it ?

Reply:
Thanks for the feedback benit, surely I have not made any change about the code and the functionality the reason behind it's because I used prettier formater to have a good design on other side no changes I made.

Reply:
@mujohn26 Kindly check if it is possible to remain with old code that you are not changing. because unnecessary changes will cause merge conflict in future.